src/Tmds.DBus.Tool/CodeGenCommand.cs

```diff
@@ -32,12 +34,13 @@ public override void Configure()
             _busOption = AddBusOption();
             _pathOption = AddPathOption();
             _norecurseOption = AddNoRecurseOption();
-            _noInternalsVisibleToOption = Configuration.Option("--no-ivt", "Don add the InternalsVisibleTo Attribute", CommandOptionType.NoValue);
+            _noInternalsVisibleToOption = Configuration.Option("--no-ivt", "Don't add the InternalsVisibleTo Attribute", CommandOptionType.NoValue);
             _namespaceOption = Configuration.Option("--namespace", "C# namespace (default: <service>)", CommandOptionType.SingleValue);
             _outputOption = Configuration.Option("--output", "File to write (default: <namespace>.cs)", CommandOptionType.SingleValue);
             _catOption = Configuration.Option("--cat", "Write to standard out instead of file", CommandOptionType.NoValue);
             _skipOptions = Configuration.Option("--skip", "DBus interfaces to skip", CommandOptionType.MultipleValue);
             _interfaceOptions = Configuration.Option("--interface", "DBus interfaces to include, optionally specify a name (e.g. 'org.freedesktop.NetworkManager.Device.Wired:WiredDevice')", CommandOptionType.MultipleValue);
+            _publicTypesOption = Configuration.Option("--public-types", "The types generated by this tool will have a public access modifier (default: internal)", CommandOptionType.NoValue);
```

Comment:
nit: rename `--public-types` to `--public`
and `The code generated by this tool will have public access modifier (default: internal).`